sep.pyx

```diff
@@ -15,7 +15,7 @@ from cpython.version cimport PY_MAJOR_VERSION
 
 np.import_array()  # To access the numpy C-API.
 
-__version__ = "1.0.3"
+__version__ = "1.0.4"
```

Comment:
You can make this 1.1.0 since this is a new feature, not just a bug fix.

Reply:
Done! (c5e3dedbd0797a2b392f899f71eed69790e711df).